trace-utils/src/trace_utils.rs

```diff
@@ -471,9 +483,15 @@ pub fn enrich_span_with_mini_agent_metadata(
         span.meta
             .insert("location".to_string(), gcp_region.to_string());
     }
+    if let Some(mini_agent_version) = &mini_agent_metadata.version {
+        span.meta.insert(
+            "_dd.mini_agent_version".to_string(),
```

Comment:
Has this tag been approved? I'm not sure if we should continue using the name `mini_agent` when this is trace agent.

Reply:
Ah, I jus read `Azure Functions, Google Cloud Functions, and Azure Spring Apps.`

But still, maybe we should switch to a better naming 🤔

Reply:
Any ideas for a better name? The packages are all named some variation of "mini agent" so at least it's consistent for now. And the tag will only be set when the environment variable `DD_MINI_AGENT_VERSION` is set, which should not be the case for Bottlecap.

Reply:
I think we need to sort this out at a later point.